src/index.js

```diff
@@ -47,10 +47,14 @@ const toJSON = (obj) => JSON.parse(JSON.stringify(obj))
 export default function (schema, opts) {
   const options = merge({}, defaultOptions, opts)
 
+  // get _id type from schema
+  options._idType = schema.tree._id.type
```

Comment:
If it does, why is there an assertion for `options._idType`? Shouldn't this always be truthy, rendering the assertion pointless?